src/plugin.video.icdrama/lib/resolvers/videobug.py

```diff
@@ -124,6 +124,17 @@ def _get_post_data(self, html, base_url):
             except Exception as e:
                 import xbmc
                 xbmc.log(str(e), xbmc.LOGERROR)
+        else:
+            try:
+                results = re.findall(r'<script.+src=".+\.vbjs\.html".+VB_TOKEN\s+?=\s+?"(.+?)".+VB_ID\s+?=\s+?"(.+?)".+(<\/script>|\/>)', html)
+                return  {
+                    'VB_ID': results[0][1],
+                    'VB_TOKEN': results[0][0],
+                    'VB_NAME': ''
+                }
+            except Exception as e:
+                import xbmc
+                xbmc.log(str(e), xbmc.LOGERROR)
```

Comment:
@aznhusband Do you know how to import the `common` module and use the `log` function in there? For debugging I always need to use the one from the `xbmc` module.